Lib/test/test_complex.py

```diff
@@ -180,6 +184,27 @@ def check(n, deltas, is_equal, imag = 0.0):
             check(2 ** pow, range(1, 101), lambda delta: False, float(i))
         check(2 ** 53, range(-100, 0), lambda delta: True)
 
+    def test_add(self):
+        self.assertEqual(1j + 1, complex(+1, 1))
+        self.assertEqual(1j + (-1), complex(-1, 1))
```

Comment:
Note that these expressions are calculated at the compile time. The first argument is a constant.

Reply:
Hmm, coverage profile shows hits...  But I'll change 2nd arguments to ``int(foo)``, that should be enough to keep BINARY_OP instruction.